### ISSUE TITLE
Binary packages for linux?

### ISSUE THREAD

**Issue by rfleschenberg** [qnzhou/PyMesh#83]
pymesh and its dependencies take quite a while to build. I wonder if it is feasible to provide binary wheels for Linux. Do you have any thoughts on this?

* https://www.python.org/dev/peps/pep-0513/
* https://github.com/pypa/manylinux

**Comment by qnzhou** [comment]
Yes, I think it would be helpful to distribute binary wheel for at least linux.  I have to admit that I struggle a bit with the python packaging tool chain because the many options available and the complexity introduced by dependencies.

Currently, my continuous integration setup builds wheel package for PyMesh automatically (see [Dockerfile](https://github.com/qnzhou/PyMesh/blob/master/docker/py2.7/Dockerfile), but it is not automatically released).  One needs to copy the wheel file out of the container (see [here](https://github.com/qnzhou/PyMesh/blob/master/docker/py2.7-slim/build.sh) for how to do it).
